package-lock.json

```diff
@@ -0,0 +1,7827 @@
+{
```

Comment:
Remove this file.. since we are using `yarn.lock`

Reply:
whoops sorry about that - all set!